project-base/tests/App/Functional/Controller/ProductRenameRedirectPreviousUrlTest.php

```diff
@@ -5,12 +5,16 @@
 namespace Tests\App\Functional\Controller;
 
 use App\DataFixtures\Demo\ProductDataFixture;
+use Faker\Provider\Text;
 use Shopsys\FrameworkBundle\Component\Domain\Domain;
-use Tests\App\Test\TransactionFunctionalTestCase;
+use Tests\App\Test\FunctionalTestCase;
+use Zalas\Injector\PHPUnit\Symfony\TestCase\SymfonyTestContainer;
 
-class ProductRenameRedirectPreviousUrlTest extends TransactionFunctionalTestCase
+class ProductRenameRedirectPreviousUrlTest extends FunctionalTestCase
```

Comment:
Any particular reason to change TransactionalFunctionalTestCase to FunctionalTestCase and injecting entityManager? If so, it would be nice to explain it in commit message

Reply:
In true, this test is one big magic for me, because when I ran it in transaction, the client request worked with old data. In debugging I didn't find why and this result is work at last. You can try it if you want :smile: